### ISSUE TITLE
API framework?

### ISSUE THREAD

**Issue by larshp** [abaplint/abaplint-cloud-foundry#1]
http://loopback.io 

https://nestjs.com

**Comment by larshp** [comment]
@sbcgua yes, I also ended up just doing it in raw code, but it would be nice with OpenAPI specifications and such

**Comment by sbcgua** [comment]
yeah openapi could be nice indeed

I checked some material on loopback and nextjs.
Seems like the later one is more trendy.
https://www.npmtrends.com/@nestjs/core-vs-loopback-vs-sails
But ... yohoho ! :)))
https://www.npmtrends.com/@nestjs/core-vs-loopback-vs-sails-vs-express

Anyway could be worth checking. It supports validation in the package and typescipt friendly... The syntax from docs looks a bit cluttered to me. But maybe we can try :)

**Comment by larshp** [comment]
yea, one step at a time

**Comment by sbcgua** [comment]
Looking through the docs of nextjs and becoming less excited. Looks quite bloated and less readable. Probably very good for bigger apps. But for a small one - light-weight express is just perfect. Just an opinion.
